File: 22237205/src/main/java/org/example/Car.java
package org.example;

class Car extends com.vehicleRental.Vehicle {
    private int passengerCapacity;

    public Car(String vehicleId, String model, double baseRentalRate, int passengerCapacity) {
        super(vehicleId, model, baseRentalRate);
        this.passengerCapacity = passengerCapacity;
    }

    @Override
    public double calculateRentalCost(int days) {
        return getBaseRentalRate() * days;
    }

    @Override
    public boolean isAvailableForRental() {
        return isAvailable();
    }

    @Override
    public String toString() {
        return "Car{" + "model='" + getModel() + '\'' + ", capacity=" + passengerCapacity + '}';
    }
}

File: Rental System 22237205/src/main/kotlin/Car.kt
package com.example.vehicle

import Vehicle

class Car(
    vehicleId: String,
    model: String,
    baseRentalRate: Double,
    private val passengerCapacity: Int
) : Vehicle(vehicleId, model, baseRentalRate) {

    override fun calculateRentalCost(days: Int): Double {
        return getBaseRentalRate() * days
    }

    override fun isAvailableForRental(): Boolean {
        return isAvailable()
    }

    override fun toString(): String {
        return "Car(model='${getModel()}', capacity=$passengerCapacity)"
    }
}